debian/control

```diff
@@ -50,6 +50,7 @@ Build-Depends: bc,
                python (>= 2.7),
                python-all-dev,
                python-nose,
+	       python-prettytable,
```

Comment:
The thing is, if it's not in build-depends, make check fails, because the target host doesn't have the package to run the tests.